.travis.yml

```diff
@@ -1,265 +1,145 @@
 # This will run on Travis' 'new' container-based infrastructure
 sudo: false
 language: c++
+dist: xenial
 
 # Blacklist
 branches:
-    only: 
-        - master
-        - coverity_scan
-    except: gh-pages
+  only: 
+    - master
+    # - coverity_scan
+  except: gh-pages
 
-# Environment variables
 env:
-    global:
-        - GH_REPO_NAME: ComPWA
-        - DOXYFILE: $TRAVIS_BUILD_DIR/doc/Doxyfile.travisCI
-        - GH_REPO_REF: github.com/ComPWA/ComPWA.git
-        - OMP_NUM_THREADS: 1
-        - LANG: en_US.UTF-8
-          #$TRAVIS_REPO_SLUG is of the form owner_name/repo_name
-        - BASEPATH: $HOME/build/$TRAVIS_REPO_SLUG/../
-          # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
-          # via the "travis encrypt" command using the project repo's public key
-        - secure: "I1LwO65P4q89f31HYnFlrlJ9yHyJ/QIAOXR5CtPfMo5FnestBtlQg6fX3p1vLNi5XdYXP4QpP51FFGJ5suz1aigC45JRaMzO98WQmMvIqYufrhkYej6lNevL0z/tHeI0aURH7N1SfSZaLwkepErssOsdjJAqP538ldA3Zz9g2gM="
+  global:
+    - GH_REPO_NAME: ComPWA
+    - DOXYFILE: $TRAVIS_BUILD_DIR/doc/Doxyfile.travisCI
```

Comment:
This Doxyfile was deleted. I think that has to be included again. Why does this have to be a global anyway?